extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -427,6 +427,9 @@ public function execute() {
 
 		if( ( $cond1 || $cond2 ) && empty( $wgDevelEnvironment ) ) {
 			$redirectUrl = WikiFactory::getLocalEnvURL( $this->mCityUrl );
+			if ( !empty($_COOKIE[\Wikia\Service\User\Auth\CookieHelper::ACCESS_TOKEN_COOKIE_NAME]) && $_SERVER['HTTP_FASTLY_SSL'] && wfHttpsAllowedForURL( $redirectUrl ) ) {
```

Comment:
Minor: This should be spacey and split with one condition per line per our coding conventions.

Reply:
Fix in: cec76f99c01